content/docs/guides/expired-branches.md

```diff
@@ -0,0 +1,297 @@
+2. A background process monitors branches and deletes them after their expiration time is reached
+3. If you reset a branch from its parent, the TTL countdown restarts using the original interval
+
+<Admonition type="important">
+Branch deletion is irreversible. Once deleted, a branch and its data cannot be recovered. All compute endpoints associated with the branch are also deleted. Deletion typically occurs within minutes of the expiration time.
+</Admonition>
+
+## Setting branch expiration
+
+You can set, update, or remove expiration timestamps through three interfaces:
+
+- **API** - Use the `expires_at` parameter with [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6) format
+- **CLI** - Use the `--expires-at` flag when creating branches or the `set-expires-at` subcommand to update
+- **Neon Console** - Use the date and time selector (automatically handles formatting)
+
+See the [Examples](#examples) section below for detailed usage of each method.
+
+## Timestamp format requirements
+
+The `expires_at` parameter must use [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6) format with second-level precision:
+
+**Format patterns:**
+
+```
+YYYY-MM-DDTHH:MM:SSZ         (UTC)
+YYYY-MM-DDTHH:MM:SS+HH:MM    (Positive UTC offset)
+YYYY-MM-DDTHH:MM:SS-HH:MM    (Negative UTC offset)
+```
+
+**Valid examples:**
+
+- `2025-07-15T18:02:16Z` (UTC)
+- `2025-07-15T18:02:16-05:00` (Eastern Standard Time)
+- `2025-07-15T18:02:16+09:00` (Japan Standard Time)
+
+**Requirements:**
+
+- Time zone is required (either `Z` or numeric offset, not both)
+- Fractional seconds are optional but only second precision is stored
+- Timestamp must be in the future
+- Maximum expiration is 1 year from the current time
+
+<Admonition type="note">
+Common errors include missing timezone (`2025-07-15T18:02:16`), past timestamps, or combining `Z` with offset (`2025-07-15T18:02:16Z-05:00`). The API will return a validation error for invalid formats.
+</Admonition>
+
+## Understanding TTL behavior
+
+When you retrieve branch information, responses include:
+
+- **`expires_at`** - The scheduled deletion timestamp
+- **`ttl_interval_seconds`** - The original time-to-live duration in seconds (read-only)
+
+The TTL interval is preserved to ensure consistent behavior when resetting branches:
+
+```json
+{
+  "branch": {
+    "id": "br-feature-67890",
+    "expires_at": "2025-07-15T18:02:16Z",
+    "ttl_interval_seconds": 86400, // 24 hours
+    "created_at": "2025-07-14T18:02:16Z"
+  }
+}
+```
+
+When you reset this branch from its parent, `expires_at` updates to the current time plus 24 hours, as 24 hours was the original interval.
+
+## Restrictions
+
+To maintain system integrity, expiration timestamps cannot be added to:
+
+- **Protected branches** - Cannot expire protected branches or protect branches with expiration
+- **Default branches** - Cannot expire default branches or set expiring branches as default
```

Comment:
What about root branches? You can have root branches and default branches. They do not need to be the same branch.
https://neon.com/docs/manage/branches#branch-types